scripts/docs/metalsmith/content/concepts.md

```diff
@@ -99,20 +112,33 @@ will be created.
 The hierarchical facets extends the concepts used in the disjunctive facetting
 in order to be able to organize the values in a tree like fashion.
 
+Idea:
+---
+Provide a simple text based menu like
+```
+- Products (200)
+  - fruits (100)
+    - citrus
+  - electronic (100)
+```
+---
+
 When using the same high-tech e-shop example (see the previous paragraph about
 [disjunctive facetting](#disjunctive-facetting)), we can imagine that TV and video
 projectors are organized in a higher category called video hardware.
 
-The structure inside the record should contain all the variations per level. For a
+Since this is a feature built on top of the Algolia API, we expect the dataset
+to follow a hierachical notation (configurable).
+
+By default, the structure inside the record should contain all the variations per level. For a
 product in the TV category, the record will have a `lvl1` attribute that will contain
 `video hardware` and a `lvl2` that will contain `video hardware > TV`. With the
 hierarchicalf feature, you can facet values per level and you will be provided the correct
 hierarchy of facet values.
 
-## Special capabilities
+## API style
```

Comment:
Global API features?

I understand we are proud of those features but for a newcomer I won't see any excitement reading this :p